Fatras/include/ActsFatras/Kernel/SimulationResult.hpp

```diff
@@ -1,34 +1,63 @@
 // This file is part of the Acts project.
 //
-// Copyright (C) 2020 CERN for the benefit of the Acts project
+// Copyright (C) 2020-2021 CERN for the benefit of the Acts project
 //
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #pragma once
 
+#include "ActsFatras/EventData/Hit.hpp"
+#include "ActsFatras/EventData/Particle.hpp"
+
+#include <iosfwd>
+#include <limits>
+#include <vector>
+
 namespace ActsFatras {
 
+/// Particle simulation status.
+enum class SimulationParticleStatus {
+  /// Particle is alive and can be simulated further.
+  eAlive,
+  /// Particle has been killed by an interaction.
+  eInteracted,
+  /// Particle has decayed.
+  eDecayed,
```

Comment:
A particle at rest has no special flag. Depending on the configured processes, there might be a lower energy/momentum cut on the outgoing particle. That defined whether the particle is considered alive or not. If it falls below such a threshold, it would be marked as `eInteracted`, i.e. not alive and killed by an interaction.